### ISSUE TITLE
MAINT: Move single cycle/diagram plotting code into new, public plotting function

### ISSUE THREAD

**Issue by nawtrey** [Becksteinlab/kda#52]
As discussed in 6. of [this comment](https://github.com/Becksteinlab/kda/pull/45#issuecomment-988461560), the single-diagram/cycle cases of `draw_cycles()` and `draw_diagrams()` probably belong elsewhere, potentially in a separate, public function. It seems a little awkward to input a single diagram/cycle into plural-named functions anyways, although a general plotting function might want to handle this case. 

This will have to be resolved after #47, and likely after #48.

**Comment by nawtrey** [comment]
I don't think this is necessary. I think it would be way more awkward to have multiple functions to handle 1 vs. 2+ diagrams. With the new `KineticModel` object, I think it will be easy to build methods for easy plotting of diagrams. I'll open a separate issue for that when the time comes, so closing this.
